ui/src/index.js

```diff
@@ -9,9 +9,9 @@ if (window.location.hostname.includes("catalogue.apprentissage.beta.gouv.fr")) {
 }
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  // <React.StrictMode>
+  <App />,
```

Comment:
In this file you may need to check the pathname to prevent the redirection in prod for the `coverage` page :) (you can check what I did for the `report` page)